.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: check-merge-conflict
+      - id: debug-statements
+      - id: trailing-whitespace
+        exclude: ^src/IAO_dicts/
```

Comment:
Did we figure out why we to exclude this folder? If trimming the whitespace breaks the parser, it seems like it's the parser's problem. (Admittedly, fixing that is out of scope for this PR!)

Reply:
Nah, it's chill

Reply:
Word